src/js/components/shepherd-element.svelte

```diff
@@ -120,6 +120,7 @@
     opacity: 0;
     outline: none;
     transition: opacity 0.3s;
+    width: 400px;
```

Comment:
This fixes this issue in mobile
<img width="348" alt="image" src="https://user-images.githubusercontent.com/706378/79281028-ced43880-7e66-11ea-9710-ece6673febb0.png">

Reply:
Hmm, I'm not sure if we want both `width` and `max-width`. Perhaps it should be `width:100%` with `max-width: 400px`?